addons/l10n_tr_nilvera_einvoice/models/account_move.py

```diff
@@ -49,6 +49,80 @@ class AccountMove(models.Model):
         copy=False,
         default='not_sent',
     )
+    l10n_tr_gib_invoice_scenario = fields.Selection(
```

Comment:
rule of thumb when adding field, use double quote for what the user see and single otherwise, i see that they follow this convention for the field above, let's do it here too :smile: (valid for all fields that you add and don't modify existing one)